packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -42,20 +145,45 @@ const SelectedEntriesTableContent = () => {
           <Table.HeaderCell fieldSchemaType="string" label="name" name="name" />
         )}
       </Table.Head>
+      <Table.LoadingBody />
       <Tbody>
-        {rows.map((entry, index) => (
-          <Tr key={entry.id}>
-            <Body.CheckboxDataCell rowId={entry.id} index={index} />
-            <Td>
-              <Typography>{entry.id}</Typography>
-            </Td>
-            {shouldDisplayMainField && (
+        {!isLoading &&
```

Comment:
I think you can remove this now if you also check the isLoading in your new Table.Body compo and use it here